src/libraries/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -619,7 +619,7 @@ public class PocoDictionary
         }
 
         [Fact]
-        public static void DictionaryOfObject_37569()
+        public static void DictionaryOfObject_NonPrimitiveTypes()
```

Comment:
While you are fixing these, can you please fix up (+ add links to) the other tests that have issue numbers?

I think keeping the "Regression" keyword in the name is fine.

DictionaryTests.cs has a whole bunch of these.

https://github.com/dotnet/runtime/blob/c4ab2a70303a58eb67651208d34d8fc6bd5d1ada/src/libraries/System.Text.Json/tests/Serialization/DictionaryTests.cs#L1668
https://github.com/dotnet/runtime/blob/c4ab2a70303a58eb67651208d34d8fc6bd5d1ada/src/libraries/System.Text.Json/tests/Serialization/DictionaryTests.cs#L1684

etc.

Reply:
Why does it matter whether we thought to add the test initially or only after a bug was discovered?

Reply:
Are you asking because of the use of the "Regression" word in the test name? I don't have a strong opinion one way or another. **I am fine with removing it** (I usually don't use that in the test name myself). I was merely suggesting that if the test name did contain it, I am also OK with leaving it in (deferring to whoever originally added it that way for why).

Reply:
> Are you asking because of the use of the "Regression" word in the test name?

Yes

Reply:
I cleaned them up. There is no more `Regression` word in the test/classes names.